ziti/edge/posture/os.go

```diff
@@ -28,12 +29,23 @@ type OsInfo struct {
 }
 
 func Os() OsInfo {
-	osType := runtime.GOOS
+	osType := "unknown"
 	osVersion := "unknown"
 
 	semVerParser := regexp.MustCompile(`^((\d+)\.(\d+)\.(\d+))`)
 
-	version, _ := host.KernelVersion()
+	_, family, version, _ := host.PlatformInformation()
+
+	if runtime.GOOS == "windows" {
```

Comment:
to lower case just in case?

Reply:
Don't this it is necessary, formally defined values.